apps/app/src/utils/pageScroll.ts

```diff
@@ -2,12 +2,12 @@ import "client-only";
 
 const classes = ["overflow-hidden"];
 
-export const lockScroll = () => {
+export const lockScroll = (addClasses: string[] = []) => {
```

Comment:
Może lepiej byłoby ukryć szczegóły implementacyjne i zamiast listy klas, przekazywać tutaj tylko parametr w stylu "mobileOnly" albo podobny?

Reply:
Muszę jeszcze trochę pomyśleć, jak to rozwiązać, ale brałem już taką opcję pod uwagę, lecz tutaj mamy problem z tym, że otwarcie / zamknięcie modala wpływa na klasy css, które są dodawane / usówane przez owrarcie / zamknięcie menu mobilnego i na odwrót. Najlepiej było by napisać funkcjonalność, która by odblokowywała pasek przewijania oraz zmieniała stan menu mobilnego na zamknięte, gdy viewport jest >= breakpointowi sm z tailwind.

Reply:
Sugestie wykorzystałem przy wdrożeniu rozwiązania problemu w #474 PR.